WcaOnRails/app/models/post.rb

```diff
@@ -2,11 +2,37 @@
 
 class Post < ApplicationRecord
   belongs_to :author, class_name: "User"
+  has_many :post_tags, autosave: true, dependent: :destroy
 
   validates :title, presence: true, uniqueness: true
   validates :body, presence: true
   validates :slug, presence: true, uniqueness: true
 
+  def reload
```

Comment:
I was thinking `expect(Post.find(post.id).tags_array).to match_array %w(wdc test)`